samples/openthread/cli/README.rst

```diff
@@ -105,7 +105,7 @@ The following snippets are available:
   Not compatible with the ``tcat`` snippet.
 
   .. note::
-    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` build target, the :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` Kconfig option has to be set to ``y``.
+    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` build target, the :makevar:`FILE_SUFFIX=ble` Kconfig option has to be set to ``y``.
```

Comment:
```suggestion
    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` board target, the :makevar:`FILE_SUFFIX=ble` Kconfig option has to be set to ``y``.
```
This is no longer a Kconfig option. How should we call it?

Reply:
It's "CMake argument" or "CMake option", as per https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/configuring_app/cmake/index.html#providing-cmake-options

```
[...] you need to set the ``ble`` variant using the :makevar:`FILE_SUFFIX` argument.
```

Also, add the following line:
```
See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information.
```